plugins/color_panel/linux/include/color_panel_plugin.h

```diff
@@ -16,23 +16,19 @@
 
```

Comment:
We should probably move to `include/color_panel/color_panel_plugin.h`. I didn't do that before because it seemed pointless since it was hard to imagine there ever being more than this one function exposed in the C API, but with an object-based C API there's no reason someone couldn't have multiple public headers, so using the normal header "namespacing" rules makes sense.